src/main/java/ReceiptSearchForm.java

```diff
@@ -0,0 +1,31 @@
+import javax.swing.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class ReceiptSearchForm {
+
+    public ReceiptSearchForm() {
+        searchButton.addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                JOptionPane.showMessageDialog(receiptSearch, "Not yet implemented");
+            }
+        });
+    }
+
+    public static void main(String[] args) {
+        JFrame frame = new JFrame("ReceiptSearchFrame");
+        frame.setContentPane(new ReceiptSearchForm().receiptSearch);
+        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+        frame.pack();
+        frame.setVisible(true);
```

Comment:
Ok here is one strategy whcih I did, create everything in PSVM in a new class. in that class you can add Receipt search form to your Jframe after your initialize it. The benefit of this is that now you are able to use this your ReceiptSearchForm to focus entirely on building the GUI, while your new class will focus on other things like when to call which Jframe. You can take a look at mine its called. RequestDrugFrame. The reason for doing this is that later on Request Drug Frame will changed to a generic GUI selecter, so it will work to  decide which GUI sholuld be displayed rather than the GUI class deciding itself. This will become more obvious when we have multiple JFrame and we need a central way to decide how to switch the view.